Prace_domowe/Praca_domowa1/Grupa1/Adam_Frej/PD1.ipynb

```diff
@@ -0,0 +1,1134 @@
+{
```

Comment:
<p>Z uwagi na to, że nigdzie wcześniej nie pojawiło się wytłumaczenie co oznaczają kolumny, należałoby to tutaj dodać - trudno rozszyfrować skrót DMC i DH.</p><p> </p><p>Generalnie należy uważać z liczeniem korelacji na zmiennych kategorycznych, nawet po zmapowaniu ich na liczby. Tutaj występowała hierarchia w danych więc takie podejście można uznać za względnie akceptowalne. Jeśli hierarchii w danych by nie było wówczas nie można liczyć korelacji (przykład będzie na kolejnych zajęciach).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>